### PR TITLE
fix spelling mistakes and grammar errors in readme

### PR DESCRIPTION
**Pull request by komanna**
Hope you like it :-)

### CODE REVIEW

**Comment by ares** [comment]
looks pretty awesome! I'll read it more thoroughly next week [test]

**Comment by komanna** [comment]
thanks, i guess there's still more to do there, but it's a start :-)

File: README.md [hunk 1]
```diff
@@ -9,36 +9,36 @@ echo "include some_modules" | puppet apply
...
 
-Note that your answers (gathered from any mode) are saved for the next run
-so you don't have to specify them again. Kafo also support default values of
-parameters so you can set only those you want to change. Also you can combine
-akk modes so you can create an answer file with default values easily
-and then use it for unattended installs.
+Note that your answers (gathered from any mode) are automatically saved for the next run
+so you don't have to specify them again. Kafo also supports default values for
+parameters so you can set only those you want to change. You can also combine
+akk modes to create an answer file with default values easily
```
Comment (ares) [inline README.md]: would you mind fixing s/akk/all/ as well please? :-)

File: README.md [hunk 2]
```diff
@@ -84,10 +84,10 @@ Usage: kafofy [options]
     -n, --name        NAME           your installer name
 ```
 
-Configuration file will be created by default template. It's the configuration
-of you installer (so you can setup log level, path to puppet modules etc).
-On the other hand, answer file must be created manually. Answer file defines
-which modules should be used and hold all values for puppet class parameters.
+The Configuration file will be created by a default template. It's the configuration
```
Comment (ares) [inline README.md]: maybe the C should not be capital after the change

File: README.md [hunk 3]
```diff
@@ -320,14 +319,14 @@ blank line.
 
 ## Hash arguments
 
-You can use Hash value not unlike Arrays. It's also multivalue type but
-you have to specify a key:value pairs like this.
+You can't use a Hash value like an Array. It's also a multivalue type but
```
Comment (ares) [inline README.md]: Actually it should have been: `You can use a Hash value similar to how you use an Array.`

File: README.md [hunk 4]
```diff
@@ -509,25 +508,25 @@ bin/foreman-installer --foreman-db-password=UNDEF
...
+You may also need to override array parameters with empty array values. For this
 purpose you can use `EMPTY_ARRAY` string as a value. Similarly you can use
 `EMPTY_HASH` for hash parameters.
 
 ## Hooks
 
-You may need to add new features to the installer. Kafo provides simple hook
-mechanism that allows you to run custom code at various time. We support
+You may need to add new features to the installer. Kafo provides a simple hook
+mechanism that allows you to run custom code at any time. We support
```
Comment (ares) [inline README.md]: `any` is maybe too strong word, we have 6 certain points in kafo run where we can hook custom code

**Comment by ares** [comment]
@komanna except few nitpicks it's ready to merge, would you mind rephrasing them and squash into one commit?

**Comment by komanna** [comment]
Thanks for your quick reply. I changed as u liked. => Ready to merge

**Comment by ares** [comment]
[test]

**Comment by domcleal** [comment]
[test]

**Comment by ares** [comment]
Thanks @komanna, merging!

**Pull request merged**